### PR TITLE
Update cocoon code for new label conventions

### PR DESCRIPTION
**Pull request by Hixie**
`team: flakes`, `severe: flake` -> `c: flake`
`needs tests` -> removed
`tech debt` -> `c: tech debt`

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM

**Comment by christopherfujino** [comment]
```
Analyzing app_dart...

  error - lib/src/request_handlers/flaky_handler_utils.dart:110:7 - Undefined name 'kSevereFlakeLabel'. Try correcting the name to one that is defined, or defining the name. - undefined_identifier

1 issue found.
```

**Comment by Hixie** [comment]
this is what you get when you're updating code by grepping the repo on your local machine and editing it on github with the web ui because you don't want to switch branches on your local machine because a bot is running out of the same repo :-D

File: app_dart/lib/src/request_handlers/flaky_handler_utils.dart [hunk 1]
```diff
@@ -15,8 +15,7 @@ import '../service/github_service.dart';
 import '../../protos.dart' as pb;
 
 // String constants.
-const String kTeamFlakeLabel = 'team: flakes';
-const String kSevereFlakeLabel = 'severe: flake';
+const String kFlakeLabel = 'c: flake';
 const String kFrameworkLabel = 'framework';
 const String kToolLabel = 'tool';
 const String kEngineLabel = 'engine';
```
Comment (keyonghan) [inline app_dart/lib/src/request_handlers/flaky_handler_utils.dart:21]: shall all these team labels be mapped to `team-*` according to https://github.com/flutter/flutter/wiki/Triage#new-team-issue-triage-process?

**Pull request merged**